tests/functional/modules/test_zos_operator_func.py

```diff
@@ -119,13 +120,31 @@ def test_zos_operator_positive_verbose_with_quick_delay(ansible_zos_module):
         assert result.get('elapsed') <= (2 * wait_time_s)
 
 
+def test_zos_operator_positive_verbose_blocking(ansible_zos_module):
+    if zoau_version_checker.is_zoau_version_higher_than("1.2.4.5"):
+        hosts = ansible_zos_module
+        wait_time_s=5
+        results = hosts.all.zos_operator(
+            cmd="d u,all", verbose=True, wait_time_s=wait_time_s
+        )
+
+        for result in results.contacted.values():
+            assert result["rc"] == 0
+            assert result.get("changed") is True
+            assert result.get("content") is not None
+            # Account for slower network
+            assert result.get('elapsed') >= wait_time_s
```

Comment:
Why was this assertion `assert result.get('elapsed') <= (2 * wait_time_s)` proposed by Demetri excluded ?

Reply:
Demetri was proposing a new test, to make sure elapsed >< wait_time_s, which you show here.  He was referring to an existing test, with < 2 * wait, which was already there.

Reply:
Here's the code he was referencing, above that change:

def test_zos_operator_positive_verbose_with_quick_delay(ansible_zos_module):
    hosts = ansible_zos_module
    wait_time_s=10
    #startmod = time.time()
    results = hosts.all.zos_operator(
        cmd="d u,all", verbose=True, wait_time_s=wait_time_s
    )
    # endmod = time.time()
    # timediff = endmod - startmod
    # assert timediff < 15

    for result in results.contacted.values():
        assert result["rc"] == 0
@@ -119,13 +120,31 @@ def test_zos_operator_positive_verbose_with_quick_delay(ansible_zos_module):
        assert result.get('elapsed') <= (2 * wait_time_s)